src/Microsoft.AspNet.Mvc.Core/Controller.cs

```diff
@@ -999,6 +999,54 @@ public virtual Task<bool> TryUpdateModelAsync<TModel>([NotNull] TModel model)
                                                                 predicate);
         }
 
+        [NonAction]
+        public Task TryValidateModelAsync(object model)
```

Comment:
how does adding `TryValidateModelAsync()` relate to adding `[ModelMetadataType]` support?